x509/src/name.rs

```diff
@@ -20,7 +22,73 @@ pub type Name<'a> = RdnSequence<'a>;
 /// ```
 ///
 /// [RFC 5280 Section 4.1.2.4]: https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.2.4
-pub type RdnSequence<'a> = alloc::vec::Vec<RelativeDistinguishedName<'a>>;
+#[derive(Clone, Debug, Default, PartialEq, Eq, Newtype)]
+pub struct RdnSequence<'a>(alloc::vec::Vec<RelativeDistinguishedName<'a>>);
```

Comment:
`Vec` is in scope:

```suggestion
pub struct RdnSequence<'a>(Vec<RelativeDistinguishedName<'a>>);
```

We should probably turn on the `unused_qualifications` lint. I'm kind of surprised it's off.

Reply:
I'll turn it on in another patch.